nmf/sparse_nmf.py

```diff
@@ -0,0 +1,172 @@
+""" Sparse Nonnegative matrix factorization
```

Comment:
Well, we will have implementations which work in Spark clusters as well as
GPU machines.
So, not sure if we want it in the sklearn branch.
For example:

https://github.com/arnlaugsson/sparkIt

Okay, will do the tests similar to the NMF function in scikits-learn.


On Sun, May 3, 2015 at 10:27 AM, Yaroslav Halchenko <
notifications@github.com> wrote:

> In nmf/sparse_nmf.py
> <https://github.com/scicubator/scikit-learn/pull/2#discussion_r29556542>:
>
> > @@ -0,0 +1,172 @@
> > +""" Sparse Nonnegative matrix factorization
>
> actually better as sklearn/decomposition/nmf
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scicubator/scikit-learn/pull/2/files#r29556542>.
>